Source/WebCore/dom/Document.h

```diff
@@ -1609,6 +1611,12 @@ class Document
     void unobserveForContainIntrinsicSize(Element&);
     void resetObservationSizeForContainIntrinsicSize(Element&);
 
+    Ref<ViewTransition> startViewTransition(RefPtr<ViewTransitionUpdateCallback> = nullptr);
+    RefPtr<ViewTransition> activeViewTransition() const;
```

Comment:
This can return a `ViewTransition*`, this is not a `protectedActiveViewTransition()` function.